src/helpers/actions/messages/replyToMessage.test.ts

```diff
@@ -230,7 +266,7 @@ describe('Cold calls', () => {
 
 		await expect(sendMessageInChannel(mockChannel, 'yo')).resolves.toBeNull();
 		expect(mockChannelSend).not.toHaveBeenCalled();
-		expect(mockLoggerError).toHaveBeenCalledOnceWith(
+		expect(mockLoggerError).toHaveBeenCalledWith(
```

Comment:
```suggestion
		expect(mockLoggerError).toHaveBeenCalledOnce();
		expect(mockLoggerError).toHaveBeenCalledWith(
```

Reply:
I wasn't sure if this was an especially important test, but if you'd like to keep it, I'll add it back.